db/seeds.rb

```diff
@@ -1,33 +1,58 @@
 # Development seed
 
-max_langs           = ENV.fetch('MAX_LANGS', 5).to_i
-max_addresses       = ENV.fetch('MAX_ADDRESSES', 5).to_i
-max_skills          = ENV.fetch('MAX_SKILLS', 10).to_i
-max_users           = ENV.fetch('MAX_USERS', 10).to_i
-max_jobs            = ENV.fetch('MAX_JOBS', 10).to_i
-max_job_users       = ENV.fetch('MAX_JOB_USERS', 10).to_i
-max_chats           = ENV.fetch('MAX_CHATS', 10).to_i
+def max_count_opt(env_name, default)
+  ENV.fetch(env_name, default).to_i
+end
+
+def log(string)
+  puts string
```

Comment:
We want to write to `STDOUT`, since we want to give feedback to the user running the rake task not only write to the regular rails log..